### DISCUSSION TITLE
LICENSE.meta file should be ignored in LICENSE file search

### DISCUSSION THREAD

**Discussion by anatawa12** [community/community#45754] [Repositories]
### Select Topic Area

Product Feedback

### Body

In my many Unity UPM Projects, LICENSE.meta file will be proceed as a license file but it's not. I think it's general issue whole issue Unity Projects so I want GitHub to solve this problem (or I want GitHub to provide way to exclude some files from license file search).

https://github.com/anatawa12/AvatarOptimizer

<img width="507" alt="image" src="https://user-images.githubusercontent.com/22656849/215404226-308f40a2-cc99-4209-92d1-1ed0dd208a57.png">

**Comment by jge162** [comment]
Hello @anatawa12 .gitignore should handle this for you. Or you looking for a different option then gitignore.

**Reply by anatawa12** [reply]
I do want to add LICENSE.meta file to git repo but ignore from license file search. So I want other solution.

**Comment by SafariMonkey** [comment]
According to [this reddit comment](https://www.reddit.com/r/Unity3D/comments/y9ji22/comment/it6oy3h/):

> If you name the file as `LICENSE.md` then Github will work correctly.
